deployment/docker/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
        TZ: Europe/Berlin
 
   policies:
-    image: docker.io/eclipse/ditto-policies:${DITTO_VERSION:-latest}
+    image: thingsdev.azurecr.io/eclipse-ditto/policies:${DITTO_VERSION:-latest}
```

Comment:
@stmaute this shouldn't be in here .. please change back

Reply:
No this shouldn't be here. I will fix it. Good catch.